services/blockchain-indexer/shared/utils/tokens.js

```diff
@@ -1,76 +0,0 @@
-/*
- * LiskHQ/lisk-service
- * Copyright © 2022 Lisk Foundation
- *
- * See the LICENSE file at the top-level directory of this distribution
- * for licensing information.
- *
- * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
- * no part of this software, including this file, may be copied, modified,
- * propagated, or distributed except according to the terms contained in the
- * LICENSE file.
- *
- * Removal or modification of this copyright notice is prohibited.
- *
- */
-const BluebirdPromise = require('bluebird');
-
-const {
-	HTTP: { request },
-} = require('lisk-service-framework');
-
-const { requestConnector, requestAppRegistry } = require('./request');
-
-const config = require('../../config');
-
-const resolveNetworkByChainID = (chainID) => {
-	const chainIDPrefix = chainID.slice(0, 2);
-	const network = config.CHAIN_ID_PREFIX_NETWORK_MAP[chainIDPrefix];
```

Comment:
`config.CHAIN_ID_PREFIX_NETWORK_MAP` can also be removed from the config file since it isn't getting used anywhere within the code.